### PR TITLE
First draft of the chart

### PR DESCRIPTION
**Pull request by bsctl**
This is a first draft of the Helm chart fo Capsule Operator.
@prometherion please review when available. Take your time! Thks

### CODE REVIEW

File: Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+sources:
+- https://github.com/clastix/capsule
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.0.0
+
+# This is the version number of the application being deployed.
+# This version number should be incremented each time you make changes to the application.
+appVersion: 0.0.0
```
Comment (prometherion) [inline Chart.yaml]: Let's put 0.0.1 although not yet released

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+# .tls
+*.crt
+*.key
+*.pem
+*.cer
+*.cnf
+
+# .kubeconfig
+*.kubeconfig
```
Comment (prometherion) [inline .gitignore]: Missing new line at the end of the file

File: Chart.yaml (hunk 1 above)
Comment (prometherion) [reply Chart.yaml]: Also, missing new line at the end of the file

**Review by prometherion — CHANGES_REQUESTED** [review]
Various missing new line at the of the file but LGTM, please, just fix the comments

File: .gitignore (hunk 2 above)
Comment (bsctl) [reply .gitignore]: @prometherion thanks for reviewing.

**Pull request merged**